db/mb_import.py

```diff
@@ -0,0 +1,33 @@
+import db
+import db.import_mb_data
+import logging
+from sqlalchemy import text
+import time
+
+SLEEP_DURATION = 30  # number of seconds to wait between runs
+
+
+def main():
+    logging.info("Checking if any import is required...")
+    while True:
+        gids_in_AB = get_new_recordings_from_AB()
+        if gids_in_AB:
+            logging.info("Updating AcousticBrainz database...")
+            db.import_mb_data.fetch_and_insert_musicbrainz_data(gids_in_AB)
+        else:
+            logging.info("No new recording found. Sleeping %s seconds." % SLEEP_DURATION)
+            time.sleep(SLEEP_DURATION)
+
+
+def get_new_recordings_from_AB():
+    with db.engine.begin() as connection:
+        query = text("""SELECT lowlevel.gid
```

Comment:
You should apply a limit to the number of recordings you import in a batch here. The value should be taken from the config files.

Reply:
I thought the same initially. The time gap of checking and importing the data for newly added recordings is 30 seconds only. Should we still add a limit here?

Reply:
I thought the same initially. As the time gap of checking the newly added recordings for import is 30 seconds only, so should we still add a limit here?

Reply:
Good point. I was thinking that what we'd do in prod is just run this script (without running `python manage.py mb_import`) and ideally it would just import all recordings over time. That would require batches here.

Reply:
Got it. Done! :baby: